lms/models/application_instance.py

```diff
@@ -80,7 +80,7 @@ class ApplicationInstance(CreatedUpdatedMixin, Base):
     organization_id = sa.Column(
         sa.Integer(), sa.ForeignKey("organization.id"), nullable=True
     )
-    organization = sa.orm.relationship("Organization")
+    organization: sa.Mapped["Organization"] = sa.orm.relationship("Organization")
```

Comment:
We'll eventually should type all the models as they are at the core of the codebase.